content/aips/58D30E766E47-AIP-100K-BAL-ACQUISITION.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
the proposal no longer needs the `<hash>-AIP` prefix.
`100K-BAL-ACQUISITION.md` is fine.